helm/templates/origins.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kraken-origin
+  labels:
+    kraken: 'true'
+spec:
+  replicas: {{ .Values.num_origins }}
+  selector:
+    matchLabels:
+      kraken-app: origin
+  template:
+    metadata:
+      labels:
+        kraken: 'true'
+        kraken-app: origin
+        kraken-peer: 'true'
+    spec:
+      containers:
+      - name: main
+        image: gcr.io/uber-container-tools/kraken-origin:v0.1.1
```

Comment:
i think its best practice to interpolate the image version

Reply:
Done.